eventsource/stores/dynamodbstore/store.go

```diff
@@ -43,11 +44,7 @@ func (store *store) LoadByAggregate(ctx context.Context, aggregateID string) (re
 
 	output, err := store.db.QueryWithContext(ctx, &input)
 	if err != nil {
-		log.
-			WithField("input", input).
-			WithField("error", err).
-			Error("Couldn't scan pages")
-		err = errors.Wrap(err, "couldn't scan pages")
+		err = fmt.Errorf("couldn't scan pages (input=%+v): %w", input, err)
```

Comment:
We should probably do the same in v2. Are you using v1?

Reply:
Turned out we were using v2. :)
Fixed that one as well.